tests/config/cpp/Issue_3236.cfg

```diff
@@ -0,0 +1,4 @@
+# Whether to fully parenthesize Boolean expressions in 'while' and 'if'
+# statement, as in 'if (a && b > c)' => 'if (a && (b > c))'.
+mod_full_paren_if_bool = true
+sp_bool                = force
```

Comment:
I still consider this a bug, that one needs to set this to `force`. The spaces are there, `sp_bool=ignore` should not remove them.